Fix #123: Update troubleshooting page to deal with: Failed to connect to raw.githubusercontent.com port 443 (#137)
oppia/oppia-web-developer-docs: Troubleshooting.md

ERROR: (dev_appserver) python2: command not found
```

> you will need to install Python 2 on your platform, even though Oppia doesn't use Python 2 anymore some parts of the App Engine dev server still need it. If you have MacOS >= 12.3, please follow the steps listed [here](<https://github.com/oppia/oppia/wiki/Installing-Oppia-(Mac-OS;-Python-3)#install-prerequisites>) to install Python 2 on your system.

Reviewer: Why was `<>` added?

Author: Removed

         from . import _imaging as core
   ImportError: cannot import name _imaging
```

> (note: [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment)

Reviewer: ```suggestion
> **Note**  
> [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment
```

it will render like this:
> **Note**  
> [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment

Author: Done

First stop the script running with Ctrl + C (or Command-period on a MAC).
Try uninstalling PIL:

```
  $ pip uninstall PIL
```

Then, uninstall and reinstall Pillow

```
  $ pip uninstall pillow
  $ pip install pillow
```

Finally, try running the `python -m scripts.start` script again.

### Command cc failed with exit status 1

> If, on MacOS Mojave V10.14.x, you get an issue arises while installing PIL library that has a lot of gibberish and that includes stuff like:

Reviewer: ```suggestion
If you encounter an issue during the installation of the PIL library on MacOS Mojave V10.14.x, which includes a lot of meaningless characters and contains logs such as:
```

Author: Done

`error: command 'cc' failed with exit status 1`

> `error: command 'clang' failed with exit status 1`

Reviewer: Make them into points

Author: Done

No Java runtime present, requesting install.
closure-compiler failed.
```

> please download [Java](https://support.apple.com/kb/DL1572?locale=en_US) and install it.

Reviewer: ```suggestion
Please download [Java](https://support.apple.com/kb/DL1572?locale=en_US) and install it.
```

Author: Done

Try running `yarn install --network-timeout 100000`, where the timeout value is in milliseconds. If that doesn’t work, try running the command with a larger network timeout value and keep increasing the timeout (e.g. adding a zero at the end of the value) until it succeeds. Then try running `python -m scripts.start` again. Note: To see if you've raised the timeout enough, you only need to run `yarn install`. You don’t need to run `python -m scripts.start` every time.

## Windows

### Windows Firewall

> It's possible that Windows firewall might be preventing localhost:8181 to launch. In such a case, you should re-config the firewall by adding new inbound rule so that ports 8181 and 8000 are allowed. (Instruction about how to add inbound rules can be found [here](<https://msdn.microsoft.com/en-us/library/hh168549(v=nav.90).aspx>))

Reviewer: Why is `<>` included in the link?

Author: Removed
